hcloud.tf

```diff
@@ -43,11 +64,86 @@ resource "hcloud_server" "k3s" {
   ssh_keys    = [ data.hcloud_ssh_key.ssh_key.id ]
   location    = "${var.location}"
 
+  public_net {
+    ipv4_enabled = true
```

Comment:
I thought we did not want to assign a public IP to the server. Technically, we can always access a server through the console, although I'm not sure sure if it is a good idea to rely on that exclusively.